runtime/compiler/env/JITServerPersistentCHTable.cpp

```diff
@@ -293,23 +295,13 @@ FlatPersistentClassInfo::serializeClass(TR_PersistentClassInfo *clazz, FlatPersi
    }
 
 std::string 
-FlatPersistentClassInfo::serializeHierarchy(TR_PersistentClassInfo *root)
+FlatPersistentClassInfo::serializeHierarchy(const JITClientPersistentCHTable *chTable)
    {
-   if (!root)
-      return "";
-
-   // walk class hierarchy to find all referenced classes
    TR::ClassTableCriticalSection serializePersistentClassInfo(TR::comp()->fe());
-   std::unordered_set<TR_PersistentClassInfo*> classes;
-   collectHierarchy(classes, root);
```

Comment:
Can we also delete this function? It looks no longer needed.

Reply:
Done